README.md

```diff
@@ -172,6 +136,66 @@ The internal concurrency structure's setup has this value as a variable.
 The actual run time of the test can be longer than specified due to the
 responses delay. Use 0 for an infinite attack.
 
+#### `-format`
+Specifies the targets format to decode.
+
+##### `json` format
+
+The JSON format makes integration with programs that produce targets dynamically easier.
+Each target is one JSON object in its own line. If present, the body field must be base64 encoded.
+
```

Comment:
This section should be expanded with the fileds of the json objects, which are mandatory and which are optional.

Reply:
+1. What do you think of checking in a JSON schema file for this? Is it over-kill?

Reply:
That's a great idea, especially with such small surface,maintaining such a file will be great for documentation and potential integrations.